pkg/controlplane/apiserver/config.go

```diff
@@ -102,6 +102,11 @@ type Extra struct {
 	SystemNamespaces []string
 
 	VersionedInformers clientgoinformers.SharedInformerFactory
+
+	// Coordinated Leader Election timers
+	LeaseDuration time.Duration
```

Comment:
nit: We have many other leases in the apiserver that these variable names are probably a bit confusing. Perhaps prepend a CLE or CoordinatedLeaderElection prefix?